README.md

```diff
@@ -67,13 +67,8 @@ Google APIs Client Libraries, in [Client Libraries Explained][explained].
 const speech = require('@google-cloud/speech');
 const fs = require('fs');
 
-// Your Google Cloud Platform project ID
-const projectId = 'your-project-id';
-
 // Creates a client
-const client = new speech.SpeechClient({
-  projectId: projectId,
-});
+const client = new speech.SpeechClient();
```

Comment:
Please note that `README.md` is auto-generated by `@google-cloud/nodejs-repo-tools` package based on existing samples. You can use the command `npm run generate-scaffolding` to regenerate it:
```sh
npm run generate-scaffolding
git commit -a -m 'regenerated scaffolding'
```
This way it will also add you to a list of contributors and probably make some other changes.

Reply:
Thanks @alexander-fenster I was not aware of that. Just did and it modified the contributors list only. I'll gladly accept to be listed although I'm not sure to deserve it :-D